python/src/pyarrow/adapters/pandas.cc

```diff
@@ -17,17 +17,18 @@
 
 // Functions for pandas conversion via NumPy
 
-#include <Python.h>
```

Comment:
It's the first include in pyarrow/adapters/pandas.h, so this should have no effect